classes/image_types_ota.bbclass

```diff
@@ -43,18 +42,14 @@ calculate_size () {
 	echo "${SIZE}"
 }
 
-export OSTREE_OSNAME
-export OSTREE_BRANCHNAME
-export OSTREE_REPO
-export OSTREE_BOOTLOADER
-
-export GARAGE_TARGET_NAME
-
-export OTA_SYSROOT="${WORKDIR}/ota-sysroot"
-
-## Common OTA image setup
-fakeroot do_otasetup () {
-	
+OTA_SYSROOT = "${WORKDIR}/ota-sysroot"
+OTA_IMAGE_ROOTFS_task-image-ota = "${OTA_SYSROOT}"
+IMAGE_TYPEDEP_ota = "ostree"
```

Comment:
Still found another one :-)

I think this should depend on `ostreecommit`.

For two reasons: First, we need the ostree native binary (which is a depends of do_image_ostreecommit), and second because we actually pull from the local repository to create the initial OTA OSTree.